### PR TITLE
Launch Supporter tier

### PR DESCRIPTION
**Pull request by davidrapson**
Changes required to make supporter tier public.

**Events / Masterclasses**

events

**Tier details now on Competitions page**

competitions

**About**

about

**Patrons**

patrons

**Pricing**

pricing

**Choose Tier**

choose-tier

**Change Tier**

change-tier

Updates to `membership-common` to follow to add Supporter to public tier list.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -10,9 +10,9 @@ object Dependencies {
   val identityCookie = "com.gu.identity" %% "identity-cookie" % "3.44"
   val identityTestUsers = "com.gu" %% "identity-test-users" % "0.5"
   val scalaUri = "com.netaporter" %% "scala-uri" % "0.4.6"
-  val membershipCommon = "com.gu" %% "membership-common" % "0.65"
-  val playGoogleAuth = "com.gu" %% "play-googleauth" % "0.2.1"
-  val contentAPI = "com.gu" %% "content-api-client" % "5.2"
+  val membershipCommon = "com.gu" %% "membership-common" % "0.67-SNAPSHOT"
```
Comment (davidrapson) [inline project/Dependencies.scala]: @rtyley Guessing I shouldn't commit this change, and update this to new version once `membership-common` PR is merged?

**Comment by davidrapson** [comment]
@rtyley Branched off PR https://github.com/guardian/membership-frontend/pull/567 as requested

**Comment by davidrapson** [comment]
// @rtyley @jennysivapalan

**Comment by davidrapson** [comment]
Dependant on https://github.com/guardian/membership-frontend/pull/567 and https://github.com/guardian/membership-common/pull/68

**Comment by rtyley** [comment]
LGTM :+1:

**Comment by davidrapson** [comment]
Confirmed and signed up as a Supporter test user on PROD

screen shot 2015-06-10 at 14 49 50

**Pull request merged**